store/interfaces.json

```diff
@@ -1,5 +1,10 @@
 {
-  "base-store-component": {
-    "component": "Example"
+  "checkout-summary": {
+    "component": "Summary",
+    "allowed": ["coupon"]
   }
-}
\ No newline at end of file
+}
+
+
+
+
```

Comment:
Please remove these empty lines, you can leave one (Python ftw) but it has 4 now.

Reply:
Done!